src/it/java/org/eclipse/microprofile/starter/utils/MPSpec.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2017-2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.eclipse.microprofile.starter.utils;
+
+/**
+ * @author Michal Karm Babacek <karm@redhat.com>
+ */
+public enum MPSpec {
+    DEFAULT(new String[][]{
+            new String[]{"data/hello", "Hello World"}
+    }),
+    CONFIG(new String[][]{
+            new String[]{"data/config/injected", "Config value as Injected by CDI Injected value"},
+            new String[]{"data/config/lookup", "Config value from ConfigProvider lookup value"}
+    }),
+    FAULT_TOLERANCE(new String[][]{
+            new String[]{"data/resilience", "Fallback answer due to timeout"}
+    }),
+    HEALTH_CHECKS(new String[][]{
```

Comment:
I thought we've changed HEATH_CHECKS to HEALTH already. Can we make sure we use HEALTH instead of HEALTH_CHECKS?

Reply:
@Emily-Jiang  @rdebusscher  We have not done it.

See master:  https://github.com/eclipse/microprofile-starter/blob/master/src/main/java/org/eclipse/microprofile/starter/addon/microprofile/servers/model/MicroprofileSpec.java#L52 

The test suite code is actually correct. I suggest to refactor it as soon as we refactor the current  code in master https://github.com/eclipse/microprofile-starter/blob/master/src/main/java/org/eclipse/microprofile/starter/addon/microprofile/servers/model/MicroprofileSpec.java#L52.